### PR TITLE
Add Sharing interface to Browser.

### PR DESCRIPTION
**Pull request by BigRoma**

### CODE REVIEW

**Review by savelichalex — APPROVED** [review]

**Review by SAnatoliiS — APPROVED** [review]

File: localization/src/languages/ko.json [hunk 1]
```diff
@@ -64,6 +64,9 @@
             "Title": "트랜잭션 확인",
             "To": "받는 사람",
             "Total": "합계"
+        },
+        "Sharing": {
+            "Share": "Share"
```
Comment (sertony) [inline localization/src/languages/ko.json:69]: Please, translate. Ask @kkkhesha for help.

File: localization/src/languages/ru.json [hunk 2]
```diff
@@ -76,6 +76,9 @@
             "AddCipherKey": "Добавить ключ шифрования",
             "CipherKeyLabel": "Ключ шифрования",
             "UseKey": "Использовать ключ"
+        },
+        "Sharing": {
+            "Share": "Поделиться"
```
Comment (sertony) [inline localization/src/languages/ru.json:81]: Please, ask @AnnStepanova to confirm the translation

File: stories/browser/src/Sharing.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                text={text}
+                status={MessageStatus.Received}
+                firstFromChain
+            />
+            <BubbleActionButton
+                type={ChatMessageType.ActionButton}
+                key="address-input-bubble-select-assets"
+                status={MessageStatus.Received}
+                text={uiLocalized.Browser.Sharing.Share}
+                onPress={onShare}
```
Comment (sertony) [inline stories/browser/src/Sharing.tsx:29]: To be honest I don't like that this is a "Share" button which allows to do anything we want...
It either should include Share logic internally or just build a customisable component from Surf itself with any button title and action for it.

@savelichalex what do you think? IMHO, the context is misleading..

Comment (savelichalex) [reply stories/browser/src/Sharing.tsx:29]: This is a consequence of browser ui and logic being split in two repos

Comment (sertony) [reply stories/browser/src/Sharing.tsx:29]: If want to keep it in UIKit, I'd make this component more generic, a.k.a `ActionWithDescription` :)

Comment (sertony) [reply stories/browser/src/Sharing.tsx:29]: Or just implement it fully from Surf. Having "Share" with not share makes no sense

**Pull request closed without merging**